<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>421</width>
    <height>378</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ClauseApp</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">background-color: rgb(20, 20, 30);</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>5</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="labelClause">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>30</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Georgia</family>
          <pointsize>15</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(230, 230, 230)</string>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelTranslation">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>30</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Georgia</family>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(230, 230, 230)</string>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelResult">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>25</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Georgia</family>
          <pointsize>13</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(230, 230, 230)</string>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineClause">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Georgia</family>
         </font>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(105, 105, 105)</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="placeholderText">
         <string>Type in your clause here.</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineTranslation">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Georgia</family>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(105, 105, 105)</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="placeholderText">
         <string>Type in the translation here.</string>
        </property>
       </widget>
      </item>
      <item alignment="Qt::AlignHCenter">
       <widget class="QPushButton" name="button0">
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="baseSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Georgia</family>
         </font>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(0, 85, 127)</string>
        </property>
        <property name="text">
         <string>PushButton</string>
        </property>
       </widget>
      </item>
      <item alignment="Qt::AlignHCenter">
       <widget class="QPushButton" name="button1">
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Georgia</family>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(0, 85, 127)</string>
        </property>
        <property name="text">
         <string>PushButton</string>
        </property>
       </widget>
      </item>
      <item alignment="Qt::AlignHCenter">
       <widget class="QPushButton" name="button2">
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Georgia</family>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(0, 85, 127)</string>
        </property>
        <property name="text">
         <string>PushButton</string>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item alignment="Qt::AlignHCenter">
       <widget class="QPushButton" name="button3">
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="baseSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Georgia</family>
          <pointsize>9</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(0, 85, 127);</string>
        </property>
        <property name="text">
         <string>PushButton</string>
        </property>
       </widget>
      </item>
      <item alignment="Qt::AlignHCenter">
       <widget class="QLabel" name="labelPoints">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>30</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Cambria Math</family>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(230, 230, 230)</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item alignment="Qt::AlignHCenter">
       <widget class="QPushButton" name="buttonNext">
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>30</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Georgia</family>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(105, 105, 105)</string>
        </property>
        <property name="text">
         <string>Next</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>421</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(20, 20, 30);
color: rgb(120, 120,120);</string>
   </property>
   <widget class="QMenu" name="menuAbout">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="font">
     <font>
      <family>Georgia</family>
      <pointsize>7</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="title">
     <string>About Me</string>
    </property>
    <property name="separatorsCollapsible">
     <bool>false</bool>
    </property>
    <addaction name="text1"/>
    <addaction name="separator"/>
    <addaction name="text2"/>
    <addaction name="text3"/>
    <addaction name="separator"/>
    <addaction name="text4"/>
    <addaction name="text5"/>
   </widget>
   <widget class="QMenu" name="menuShortcuts">
    <property name="font">
     <font>
      <family>Georgia</family>
      <pointsize>7</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Keybinds</string>
    </property>
    <addaction name="actionExit"/>
    <addaction name="actionButton0"/>
    <addaction name="actionButton2"/>
    <addaction name="actionButton1"/>
    <addaction name="actionButton3"/>
    <addaction name="actionButtonNext"/>
   </widget>
   <widget class="QMenu" name="menuAdd_clause">
    <property name="font">
     <font>
      <family>Georgia</family>
      <pointsize>7</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Add clause</string>
    </property>
    <addaction name="actionShow"/>
    <addaction name="actionReturn"/>
   </widget>
   <addaction name="menuShortcuts"/>
   <addaction name="menuAbout"/>
   <addaction name="menuAdd_clause"/>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(20, 20, 30);</string>
   </property>
  </widget>
  <action name="actionNext">
   <property name="text">
    <string>Next</string>
   </property>
  </action>
  <action name="text1">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Simple language learning app.</string>
   </property>
   <property name="iconText">
    <string>Simple language learning app.</string>
   </property>
   <property name="toolTip">
    <string>Simple language learning app.</string>
   </property>
   <property name="font">
    <font>
     <family>Georgia</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
   <property name="shortcutVisibleInContextMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="text2">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Select your answer and press next</string>
   </property>
   <property name="font">
    <font>
     <family>Georgia</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
   <property name="shortcutVisibleInContextMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="text3">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>to proceed to the next question.</string>
   </property>
   <property name="font">
    <font>
     <family>Georgia</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
   <property name="shortcutVisibleInContextMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionquestion_Use_escape_to">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>question.</string>
   </property>
   <property name="font">
    <font>
     <family>Georgia</family>
     <pointsize>11</pointsize>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="visible">
    <bool>true</bool>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
   <property name="shortcutVisibleInContextMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionExit">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="font">
    <font>
     <family>Georgia</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="shortcut">
    <string>Esc</string>
   </property>
  </action>
  <action name="actionButton0">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Select answer 1</string>
   </property>
   <property name="font">
    <font>
     <family>Georgia</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="shortcut">
    <string>1</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::WindowShortcut</enum>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
   <property name="shortcutVisibleInContextMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionButton1">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Select answer 2</string>
   </property>
   <property name="font">
    <font>
     <family>Georgia</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="shortcut">
    <string>2</string>
   </property>
  </action>
  <action name="actionButton2">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Select answer 3</string>
   </property>
   <property name="font">
    <font>
     <family>Georgia</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="shortcut">
    <string>3</string>
   </property>
  </action>
  <action name="actionButton3">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Select answer 4</string>
   </property>
   <property name="font">
    <font>
     <family>Georgia</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="shortcut">
    <string>4</string>
   </property>
  </action>
  <action name="actionButtonNext">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Proceed</string>
   </property>
   <property name="font">
    <font>
     <family>Georgia</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="shortcut">
    <string>N</string>
   </property>
  </action>
  <action name="actionShow">
   <property name="text">
    <string>Show form</string>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>9</pointsize>
    </font>
   </property>
  </action>
  <action name="actionReturn">
   <property name="text">
    <string>Go back</string>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>9</pointsize>
    </font>
   </property>
  </action>
  <action name="text4">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Add clause is clickable.</string>
   </property>
   <property name="font">
    <font>
     <family>Georgia</family>
     <pointsize>11</pointsize>
    </font>
   </property>
  </action>
  <action name="text5">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>It toggles a text form.</string>
   </property>
   <property name="font">
    <font>
     <family>Georgia</family>
     <pointsize>11</pointsize>
    </font>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
